contentcuration/contentcuration/static/js/edit_channel/file_upload/views.js

```diff
@@ -503,31 +496,42 @@ var FormatSlot = BaseViews.BaseListNodeItemView.extend({
     template: require("./hbtemplates/format_item.handlebars"),
     dropzone_template : require("./hbtemplates/format_dropzone_item.handlebars"),
     upload_in_progress:false,
+    collapsedClass: "glyphicon-menu-up",
+    expandedClass: "glyphicon-menu-down",
+
+    getToggler: function () { return this.$("#format_item_" + this.model.id); },
+    getSubdirectory: function () {return this.$("#format_item_" + this.model.id +"_sub"); },
 
     'id': function() { return "format_slot_item_" + this.model.get("id"); },
     selector: function() { return this.model.get("id") + "_"  + this.node.get('id') + "_dropzone" + ((this.file)? "_swap" : "") },
     className:"row format_editor_item",
     initialize: function(options) {
-        _.bindAll(this, 'remove_item','file_uploaded','file_added','file_removed','file_failed', 'create_dropzone', 'set_file');
+        _.bindAll(this, 'remove_item','file_uploaded','file_added','file_removed','file_failed', 'create_dropzone', 'set_file', 'set_uploading');
         this.containing_list_view = options.containing_list_view;
+        this.content_node_view = options.content_node_view;
+        this.language_view = options.language_view;
         this.file = options.file;
         this.originalFile = this.file;
+        this.files = options.files;
+        this.collection = options.collection;
         this.allow_edit = options.allow_edit;
         this.node = options.node;
         this.render();
     },
     events: {
-        'click .format_editor_remove ' : 'remove_item'
+        'click .format_editor_remove ' : 'remove_item',
+         "click .multilanguage_folder" : "toggle"
     },
     render: function() {
         this.$el.html(this.template({
             file: (this.file)? this.file.toJSON() : null,
             preset: this.model.toJSON(),
             selector: this.selector(),
+            multi_language: this.model.get('multi_language') && !this.file,
             allow_edit: this.allow_edit,
             src: (this.file)? this.file.get('storage_url') : null
```

Comment:
You could probably [short circuit evaluate](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#Short-circuit_evaluation) things like this instead of using ternary, but that's stylistic preference